bundles/org.openhab.binding.sagercaster/README.md

```diff
@@ -0,0 +1,81 @@
+# Sager Weathercaster Binding
+
+The Sager Weathercaster is a scientific instrument for accurate prediction of the weather. 
+
+## Limitations
+
+* To operate, this binding will need to use channel values provided by other means (e.g. Weather Binding, Netatmo, a 1-Wire personal weather station...)
+
+* This binding buffers readings for some hours before producing weather forecasts(wind direction and sea level pressure). SagerWeatherCaster needs an observation period of minimum 6 hours.
+
+For these reasons, this binding is not a binding in the usual sense.
+
+## Discovery
+
+A default `sagercaster` thing will be automatically discovered, based on system location.
+
+## Binding Configuration
+
+The binding itself does not require any configuration.
+
+## Thing Configuration
+
+### SagerCaster
+
+| Name               | Type     | Description                                                              |
+|--------------------|----------|--------------------------------------------------------------------------|
+| location           | Location | Latitude and longitude of the desired weather forecast.                  |
+| observation-period | int      | Minimum delay (in hours) before producing forecasts. Defaulted to 6.     |
+
+## Channels
+
+The binding will use some input channels, that can be configured directly with profiles (sample below).
+
+| Name               | Type          | Description                                                     |
+|--------------------|---------------|-----------------------------------------------------------------|
+| is-raining         | Switch        | On if it is raining, else Off.                                  |
+| rain-qtty          | Number        | Any value that give indication of a current rain volume         |
+|                 or | Number:Speed  | Any value that give indication of a current rain volume eg mm/h |
+|                 or | Number:Length | Any value that give indication of a current rain volume eg mm   |
+|                    |               |                                                                 |
```

Comment:
Should add the other input channels to this table as well. Then maybe have a separate table for output channels, or just make another column to indicate whether the channel is an input or output.

Reply:
Done